src/providers/theme/theme.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@angular/core';
+import { StatusBar } from '@ionic-native/status-bar';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
```

Comment:
Maybe we could remove it, since the value doesn't need a history. What do you think?

Reply:
done. Thanks!